spec/controllers/reports/regions_controller_spec.rb

```diff
@@ -258,8 +258,8 @@ def refresh_views
     it "returns period info for current month" do
       today = Date.current
       Timecop.freeze(today) do
-        patient = create(:patient, registration_facility: @facility, recorded_at: today)
-        create(:blood_pressure, :under_control, recorded_at: today, patient: patient, facility: @facility)
+        patient = create(:patient, registration_facility: @facility, recorded_at: 2.months.ago)
+        create(:bp_with_encounter, :under_control, recorded_at: Time.current.yesterday, patient: patient, facility: @facility)
```

Comment:
Bunch of specs need the smarter fixture that includes an encounter for things to work w/ the new views.